examples/collection-only/collection.json

```
@@ -201,20 +201,20 @@
   "links": [
     {
       "rel": "self",
-      "href": "https://storage.cloud.google.com/earthengine-test/catalog/COPERNICUS_S2.json"
+      "href": "https://raw.githubusercontent.com/radiantearth/stac-spec/examples-refresh/examples/collection-example-1/collection.json"
```

This link will be invalid after the merge.

---

Yes, I have a todo on that for 'after merge'. Though I'm also contemplating making this a more self-contained one.

---

Why not do it in this PR? We can predict the future URL from the tags. 

The URL after release will be:

https://github.com/radiantearth/stac-spec/blob/v1.0.0-beta.2/extensions/README.md
```suggestion
      "href": "https://raw.githubusercontent.com/radiantearth/stac-spec/v1.0.0-rc.1/examples/collection-example-1/collection.json"
```